easycord/plugins/levels.py

```
@@ -75,6 +79,78 @@ def get_entry(self, guild_id: int, user_id: int) -> dict:
+            title=f"📊 {ctx.guild.name} Ranks",
+            description="\n".join(lines),
+            color=discord.Color.blurple(),
+        )
+
+    async def _set_config_value(self, guild_id: int, section: str, key: int, value) -> None:
+        def updater(config: dict) -> None:
+            config.setdefault(section, {})[str(key)] = value
+
+        await self._store.update_config(guild_id, updater)
+
+    async def _remove_config_value(self, guild_id: int, section: str, key: int):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_remove_config_value`

(ANN202)